src/main/webapp/3Dscript.js

```diff
@@ -1,29 +1,32 @@
-import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+// import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
```

Comment:
should comment be removed?

Reply:
+1
For the client code that not be able to add unit test, you can test it manually to make sure the code can run and doesn't contain bugs.